src/approaches/nsrt_learning_approach.py

```diff
@@ -53,12 +54,90 @@ def learn_from_offline_dataset(self, dataset: Dataset) -> None:
 
     def _learn_nsrts(self, trajectories: List[LowLevelTrajectory],
                      online_learning_cycle: Optional[int]) -> None:
+        dataset_fname, _ = utils.create_dataset_filename_str(
+            saving_ground_atoms=True,
+            online_learning_cycle=online_learning_cycle)
+        # If CFG.load_atoms is set, then try to create a GroundAtomTrajectory
+        # by loading sets of GroundAtoms directly from a saved file.
+        if CFG.load_atoms:
+            os.makedirs(CFG.data_dir, exist_ok=True)
+            # Check that the dataset file was previously saved.
+            if os.path.exists(dataset_fname):
+                # Load the ground atoms dataset.
+                with open(dataset_fname, "rb") as f:
+                    ground_atom_dataset_atoms = pkl.load(f)
+                assert len(trajectories) == len(ground_atom_dataset_atoms)
+                logging.info("\n\nLOADED GROUND ATOM DATASET")
+
+                if CFG.env == "behavior":  # pragma: no cover
+                    pred_name_to_pred = {
+                        pred.name: pred
+                        for pred in self._get_current_predicates()
+                    }
+                    new_ground_atom_dataset_atoms = []
+                    # Since we save ground atoms for behavior with dummy
```

Comment:
if you're loading atoms from disk, why do you actually care if the classifiers are correct? where would they be used?

Reply:
So we don't actually care for the purposes of NSRT learning - I just wanted to set them back to be correct so that if in the future we use these atoms for something we don't get completely bamboozled by why they're horribly wrong

Reply:
yeah, +1 to what Nishanth said